linux_os/guide/system/kernel_build_config/gcc_plugin/kernel_config_gcc_plugin_randstruct/rule.yml

```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+prodtype: fedora,rhel9
+
+title: 'Randomize layout of sensitive kernel sttructures'
```

Comment:
```suggestion
title: 'Randomize layout of sensitive kernel structures'
```

Reply:
This is a repeated typo, :stuck_out_tongue: